hs_file_types/models/model_instance.py

```diff
@@ -332,18 +345,6 @@ def add_sub_element_value_triple(sub_element_node, sub_element_value):
         if self.logical_file.metadata_schema_json:
             model_title = self.logical_file.metadata_schema_json.get('title', "")
         graph.add((model_meta_node, DC.title, Literal(model_title)))
-        if self.has_model_output:
```

Comment:
Ok, so the issue occurred because these terms were set after the namespace manager bind?